src/cpu/test.rs

```diff
@@ -814,6 +831,79 @@ fn test_shift() {
     assert_eq!(cpu.get_flag(Flag::H), false);
     assert_eq!(cpu.get_flag(Flag::C), true);
 
+    cpu.set_reg(Register8Bit::B, 0x95);
```

Comment:
Can you add comments before each test to indicate which instruction its testing?

Reply:
https://tenor.com/view/bruh-gpo-bad-trade-nty-bro-gif-24142673